mcstatus/pinger.py

```diff
@@ -222,10 +224,15 @@ def __init__(self, raw):
 
     players: Players
     version: Version
-    description: str
+    motd: Motd
     favicon: Optional[str]
     latency: float = 0
 
+    @property
+    @deprecated(replacement="motd field")
```

Comment:
Would be nice if someone propose here date.

Reply:
We could make the new class a descriptor, returning string on `__get__` which will be deprecated, and instead you're expected to call some method, such as `plain` on that descriptor. This is pretty complex and I generally don't like too much complexity in a code-base, but since this is just for deprecation, and will be removed after some period it's probably ok

Reply:
Okay, I will try this. But how about date?

Reply:
No idea, that can be decided on later, maybe add a TODO into the PR's body, or into the code as comment so it's not forgotten

Reply:
> returning string on `__get__`

This will introduce new problems:
```py
>>> class Motd:
...     a = "a"
...     def __get__(self, instance, owner):
...         return "b"
... 
>>> Motd().a
'a'
>>> class Test:
...     b = Motd()
... 
>>> Test().b
'b'
>>> Test().b.a
Traceback (most recent call last):
  ...
AttributeError: 'str' object has no attribute 'a'

Reply:
Ah, yeah that does make sense. In that case we could either make this a string subclass with added logic on `__init__`, or break backwards compatibility which probably isn't viable.

Reply:
This would be a Frankenstein, not an independent class (which actually breaks `The Single Responsibility Principle` from SOLID).

But I agree, just a breaking change isn't viable.

Reply:
Good idea will be wait for #306, what I actually will do.

@ItsDrike add please `do-not-merge` label.